cards/cards.css

```diff
@@ -0,0 +1,76 @@
+.disp-card {
```

Comment:
I feel a better class name would be needed for this

Reply:
Actually, I've used a common naming style for displaying items as `disp-componentName`. Anyways, I respect you strong opinion.